File: Brainz/Model/Alias.cs
namespace brainz.Model
{
    using Newtonsoft.Json;

    public class Alias
    {
        [JsonProperty("sort-name")]
        public string SortName { get; set; }

        [JsonProperty("name")]
        public string Name { get; set; }

        [JsonProperty("locale")]
        public object Locale { get; set; }

        [JsonProperty("type")]
        public object Type { get; set; }

        [JsonProperty("primary")]
        public object Primary { get; set; }

        [JsonProperty("begin-date")]
        public object BeginDate { get; set; }

        [JsonProperty("end-date")]
        public object EndDate { get; set; }
    }
}

File: src/model/alias.go
package model

type Alias struct {
	SortName  string `json:"sort-name"`
	Name      string `json:"name"`
	Locale    string `json:"locale"`
	Type      string `json:"type"`
	Primary   bool   `json:"primary"`
	BeginDate string `json:"begin-date"`
	EndDate   string `json:"end-date"`
}
